pkg/semtech-loramac/patches/0001-add-RIOT-makefiles.patch

```diff
@@ -0,0 +1,57 @@
+From d45cbc53687d090a69cf669e8e8a9fb5460db943 Mon Sep 17 00:00:00 2001
```

Comment:
Please don't use a patch here. Put the Makefiles into the package directory and just copy them over in the compilation step (see how e.g., heatshrink does it). Way easier to maintain.